src/BMI.ts

```diff
@@ -7,7 +7,10 @@
  */
 function calculateBMI(weight: number, height: number): number {
   // Write your code here 👇🏼
-  const BMI = 0;
+
+  let BMI = 0;
+  height = (height /100);
+  BMI = (weight/(height^2));
```

Comment:
`x^2` in JavaScript is not the square. To square a number use `x ** 2` or `Math.pow(x, 2)`